src/pages/Overview/OverviewToolbar.tsx

```diff
@@ -111,10 +111,17 @@ export class OverviewToolbar extends React.Component<Props, State> {
     this.setState({ isSortAscending: newDir });
   };
 
-  updateOverviewType = (otype: OverviewType) => {
-    HistoryManager.setParam(URLParam.OVERVIEW_TYPE, otype);
-    this.setState({ overviewType: otype });
-    this.props.onRefresh();
+  updateOverviewType = (otype: String) => {
```

Comment:
can't we enforce type at compile time as before?

Reply:
We can't because `handleSelect` takes `(_: String) => any` :(
We can either cast it to the type and ignore the error, or do more complete error handling (which is what I decided to do).

Reply:
Ok, I see. Maybe we could make `ToolbarDropdown` use generics, but let's keep that for later :)